include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -7124,17 +7124,22 @@ FEFaceEvaluation<dim, fe_degree, n_q_points_1d, n_components_, Number>::
                   const bool        evaluate_gradients)
 {
   const unsigned int side = this->face_no % 2;
-  const unsigned int dofs_per_face =
+
+  const unsigned int static_dofs_per_face =
```

Comment:
while there, this is `constexpr`, right?

Reply:
Yes, I will change the other places as well for consistency.